src/sentry/utils/assets.py

```diff
@@ -32,17 +64,20 @@ def get_frontend_app_asset_url(module: str, key: str) -> str:
     if not key.startswith("entrypoints/"):
         raise AssertionError(f"unexpected key: {key}")
 
-    entrypoints, key = key.split("/", 1)
+    asset_path, key = key.split("/", 1)
     versions = _frontend_versions()
-    if versions:
-        entrypoints = "entrypoints-hashed"
-        key = versions[key]
+
+    # When a frontend entrypoint versions config is provided use to map the
+    # asset file to a hashed entrypoint
+    if versions and key in versions["entrypoints"]:
+        asset_path = "entrypoints-hashed"
+        key = versions["entrypoints"][key]
```

Comment:
this changes behaviour.  it *should* crash if things are in an invalid state rather than silently passing